kornia/geometry/liegroup/se3.py

```diff
@@ -248,6 +248,25 @@ def inverse(self) -> 'Se3':
         r_inv = self.r.inverse()
         return Se3(r_inv, r_inv * (-1 * self.t))
 
+    @classmethod
+    def random(cls, batch_size: Optional[int] = None, device=None, dtype=None) -> 'Se3':
+        """Create a Se3 group representing a random transformation.
+
+        Args:
+            batch_size: the batch size of the underlying data.
+
+        Example:
+            >>> s = Se3.random()
+            >>> s = Se3.random(batch_size=3)
+        """
+        r = So3.random(batch_size, device, dtype)
```

Comment:
```suggestion
        r = So3.random(batch_size, device, dtype)
        shape: Tuple[int, ...]
```
i think with this mypy won't complain